.github/utils/validate_input_templates.py

```diff
@@ -0,0 +1,46 @@
+import glob
+import json
+
+import boto3
+import logging
+
+log_format = "[%(levelname)s] %(message)s"
+logging.basicConfig(level=logging.INFO, format=log_format)
+
+
+def is_s3_path(value):
+    return isinstance(value, str) and value.startswith('s3://') and '<' not in value
+
+
+def s3_path_exists(s3_client, s3_path):
+    try:
+        bucket, key = s3_path.replace("s3://", "").split("/", 1)
+        s3_client.head_object(Bucket=bucket, Key=key)
+        return True
+    except Exception as e:
+        return False
+
+
+def validate_json_files(json_files):
+    s3_client = boto3.client('s3')
+    all_paths_valid = True
+
+    for json_file in json_files:
+        with open(json_file, 'r') as file:
+            data = json.load(file)
+            for key, value in data.items():
```

Comment:
I think this won't catch nested fields, like here for example: https://github.com/Ultimagen/healthomics-workflows/blob/e46fd88cc230dcacf5af3f43ad37d62b7cd78b16/workflows/efficient_dv/input_templates/efficient_dv_template-WGS-germline-v1_2_rc2--lt20mer-30X-50X-.json#L10